src/pages/not-found-page/not-found-page.tsx

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+import { AppRoute } from '../../app/routes';
+
+function NotFoundPage(): JSX.Element {
+  const containerStyle: React.CSSProperties = {
```

Comment:
You can have styles here. But I'd rather moved them to separate file within the same folder. Your code will look neater